design/frontend/template/autocomplete/category.phtml

```diff
@@ -1,37 +1,26 @@
-<?php
-
-$image_base_url = Mage::getBaseUrl();
-
-?>
-
 <!-- Category hit template -->
 <script type="text/template" id="autocomplete_categories_template">
     <a class="algoliasearch-autocomplete-hit" href="{{url}}">
         {{#image_url}}
-        <div class="thumb">
-            <img src="<?php echo $image_base_url; ?>{{image_url}}" />
-        </div>
+            <div class="thumb">
+                <img src="{{image_url}}" />
```

Comment:
It would be nice to be able to use the category thumbnail image for this, since category images cannot be resized and are therefor many times larger than the small size needed for the category suggestions.

It is not possible however to add these in the configuration so I had to add these with code (add attribute 'thumbnail' to loaded category collection).
It would be nice if the thumbnail field was fetched by default or if there was an option to add it to the indexed fields.